test/unit/specs/components/Identity/createIdSyncs.spec.js

```diff
@@ -32,7 +32,7 @@ describe("Identity::createIdSyncs", () => {
     }, {});
   };
 
-  it("tracks id syncs", done => {
+  it("tracks id syncs", () => {
```

Comment:
Are you sure this change should be made? I don't think this will run the `expect` without the `done`. If `idSyncs.process` returned a promise, we wouldn't have to do this `setTimeout` business at all.

It looks like you left an `fdescribe` in as well.